realm/realm-library/src/androidTest/kotlin/io/realm/DynamicMixedTests.kt

```diff
@@ -0,0 +1,128 @@
+package io.realm
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import io.realm.rule.TestRealmConfigurationFactory
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.junit.runner.RunWith
+import java.util.*
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+
+// FIXME: MIXED PARAMETRIZED TESTS FOR INDEXED AND UNINDEXED
+@RunWith(AndroidJUnit4::class)
+class DynamicMixedTests {
+    @get:Rule
+    val configFactory = TestRealmConfigurationFactory()
+
+    @Rule
+    @JvmField
+    val folder = TemporaryFolder()
+
+    init {
+        Realm.init(InstrumentationRegistry.getInstrumentation().targetContext)
+    }
+
+    @Test
+    fun writeRead_primitive() {
+        val realm = DynamicRealm.getInstance(configFactory.createConfiguration("Mixed"))
```

Comment:
I would either pull these up in `setUp/tearDown` methods or use `realm.use { }`, otherwise the Realm instance might not be closed in case of unit test failures.